File: client/src/_models/article.ts
import { ArticleImages } from "./articleImages";
import { Store } from "./store";

export interface Article {
    id : string;
    href: string;
    name : string;
    price : number;
    gender: string;
    type: string;
    imgSources : ArticleImages[];
    storeId: Store;
}

File: API/Entitites/Article.cs
using System.Collections.Generic;

namespace API.Entitites
{
    public class Article
    {
        public int Id { get; set; }
        public string href { get; set; }
        public string name { get; set; }
        public int price { get; set; }
        public char gender { get; set; }
        public string type { get; set; }
        public ICollection<ArticleImages> imgSources { get; set; }
        public int storeId { get; set; }
        public Store store;
    }
}